react/react2/week/github-user/src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Header from './Header';
+import Footer from './Footer';
+import GitUserContext from './GitUserContext';
```

Comment:
Context isn't a component is just a way to pass the data through your components. 
```suggestion
import GitUserFetch from './GitUserFetch';
```

Reply:
I have updated according to feedback.